### PR TITLE
[FIX] 14.0 variant price computation if original_price_unit = 0

### PR DESCRIPTION
**Pull request by PierrickBrun**
pre-commit made me remove modules from the "non-installable" list but I guess it should be done somewhere else

### CODE REVIEW

**Comment by rousseldenis** [comment]
> pre-commit made me remove modules from the "non-installable" list but I guess it should be done somewhere else

Are you based on 14.0 ? as partner_contact_company for instance is not installable (10.0)

**Comment by PierrickBrun** [comment]
> > pre-commit made me remove modules from the "non-installable" list but I guess it should be done somewhere else
> 
> Are you based on 14.0 ? as partner_contact_company for instance is not installable (10.0)

I've just discovered that `git sparse-checkout` (https://www.git-scm.com/docs/git-sparse-checkout) doesn't play well with our pre-commit config

**Review by kevinkhao — APPROVED** [review]
LGTM

File: shopinvader/models/shopinvader_variant.py [hunk 1]
```diff
@@ -257,7 +257,7 @@ def _get_price(self, qty=1.0, pricelist=None, fposition=None, company=None):
                 )
             # Compute discount
             price_dp = self.env["decimal.precision"].precision_get("Product Price")
-            if float_compare(
+            if original_price_unit != 0 and float_compare(
```
Comment (rousseldenis) [inline shopinvader/models/shopinvader_variant.py]: You can use float_is_zero for this purpose

**Review by hparfr — APPROVED** [review]

**Pull request merged**